kw/platform/wsgi.py

```diff
@@ -0,0 +1,78 @@
+import re
+import time
+from datetime import datetime
+
+import webob
+from dateutil.parser import parse
+from webob.dec import wsgify
+
+from . import settings
+
+
+user_agent_re = re.compile(settings.VALID_USER_AGENT_RE)
+req_slowdown_datetime = parse(settings.REQUESTS_SLOWDOWN_DATETIME)
+req_restrict_datetime = parse(settings.REQUESTS_RESTRICT_DATETIME)
+
+
+def _refuse_request(req, app):
+    return webob.exc.HTTPBadRequest("Invalid User-Agent")
```

Comment:
I'd add a note that we expect it to be KW-RFC-22 compliant.

Reply:
I had the note there but than I thought this is a public repository, if someone who doesn't work for kiwi.com used this library, it wouldn't make much sense to have it there as the RFC isn't publicly available.

Reply:
Well, that's fine, we're here to serve Kiwi.com and not the rest of the world — we're just coincidentally doing it out in the open 🙃 

Buuuut actually I'm already working on publishing the RFC publicly as well, so it will make more sense after that :D